docs/2.deploy/20.providers/coolify.md

```diff
@@ -0,0 +1,38 @@
+# Coolify
+
+> Deploy Nitro apps to [Coolify](https://coolify.io).
+
+**Preset:** `coolify`
+
+:read-more{title="coolify.io" to="https://coolify.io/docs/"}
+
+::note
+This is the recommended preset for Coolify deployments.
+::
+
+Coolify supports deploying both static and server-side rendered apps with zero configuration.
+
+## Set up your web app
+
+In your project, set Nitro preset to `coolify`.
```

Comment:
Instead, we should instruct the build commands to use `SERVER_PRESET=coolify` environment variable.